app/views/users/show.html.erb

```diff
@@ -0,0 +1,104 @@
+<% if current_user.status %>
+<div class="px-6 y py-2">
+  <div class="text-justify py-4 mt-2.5">
+    <%= render "shared/header_stamp" %>
+    <h3>Mi cuenta</h3>
+  </div>
+
+  <div class="grid lg:grid-cols-2 md:grid-cols-1 mt-2.5">
+    <div clas="lg:col-start-1 lg:col-end-2">
+      <h5>Nombre completo</h5>
+      <p><%= current_user.name %></p>
+    </div>
+
+    <div clas="col-start-2 col-end-3">
+      <h5>Correo electrónico</h5>
+      <p><%= current_user.email %></p>
+    </div>
+  </div>
+
+  <div class="text-justify py-4 mt-2.5">
+    <h3>Mis negocios</h3>
+  </div>
+  
+  <hr class="w-full border-scene-neutral">
+
+       
+  <% MarketPlace.where(user_id: current_user.id).each do |market| %>
+    <h5 class="mt-6">Nombre</h5>
+    <p><%= market.business_name %></p>
+      
+    <div class="grid lg:grid-cols-2 md:grid-cols-1 mt-4">
+      <div clas="lg:col-start-1 lg:col-end-2">
+        <h5>Direccion</h5>
+        <p><%= market.address %></p>
+      </div>
+
+      <div clas="col-start-2 col-end-3">
+        <h5>Teléfono:</h5>
+        <p><%= User.find(market.user_id).phone %></p>
+      </div>
+    </div>
+
+    <div class="grid lg:grid-cols-2 lg:grid-rows-4 md:grid-cols-1">
+      <div class="mt-4">
+        <h5>Giro</h5>
+        <p><%= market.business_line%></p>
+      </div>
+      
+      <div class="mt-4">
+        <h5>Vendiendo</h5>
+        <p><%= market.selling %></p>
+      </div>
+
+      <div class="mt-4">
+        <h5>Forma(s) de pago</h5>
+        <p><%= market.payment %></p>
+      </div>
+      
+      <div class="mt-4">
+        <h5>Tipo(s) de entrega</h5>
+        <p><%= market.pickup_type%></p>
+      </div>
+
+      <div class="mt-4">
+        <h5>Servicio a domicilio</h5>
+        <p><%= market.service_days %></p>
+      </div>
+      
+      <div class="mt-4">
+        <h5>Horario</h5>
+        <p><%= market.open_hour.strftime('%H:%M') %> a <%= market.close_hour.strftime('%H:%M') %></p>
+      </div>
+    </div >
+
+    <div class="flex flex-row items-center">
+      <%= button_to 'Eliminar', market_place_path(market.id), method: :delete, data: { confirm: 'Are you sure?' }, class: 'button btn-neutral mt-4 w-40 justify-center' %>
+      <%= button_to 'Editar', edit_market_place_path(market.id), method: 'get', class: 'button btn-sec ml-2 mt-4 w-40 justify-center' %>
+    </div>
+
+    <!-- End Grid Mis negocios-->
+    <hr width="90%" class="w-full mt-4 border-scene-neutral">
+  <%end%>
+
+  <!-- ---------------------   End Market Places --------------------------- -->
+</div>
+
+<div align="center" class="mb-16">
+  <%= button_to 'Añadir nuevo negocio', new_market_place_path(MarketPlace.where(user_id: current_user.id).select("id")), method: 'get', class: 'button btn-prim w-80 justify-center' %>
```

Comment:
Why do we need to pass an Id to this path?

Reply:
Also, if possible, remove the query

Reply:
Removed 👍